CHANGELOG.md

```diff
@@ -77,6 +77,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Bug Fixes
 
+* (x/auth) [#18746](https://github.com/cosmos/cosmos-sdk/pull/18746) Fixed duplicated `vesting` subcommand of `<appd> tx --help` and missing `<appd> tx auth update-params-proposal` subcommand.
```

Comment:
Please remove the changelog. This issue never got released.

Reply:
Done! Just saw the ci error and added the changelog...